Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -281,7 +288,13 @@ namespace gdjs {
       this.gravityScale = behaviorData.gravityScale;
       this.layers = behaviorData.layers;
       this.masks = behaviorData.masks;
-      this.currentContacts = this.currentContacts || [];
+      this.contactsStartedBetweenFrames =
+        behaviorData.contactsStartedBetweenFrames || [];
+      this.contactsStartedBetweenFrames.length = 0;
+      this.contactsEndedBetweenFrames =
+        behaviorData.contactsEndedBetweenFrames || [];
+      this.contactsEndedBetweenFrames.length = 0;
+      this.currentContacts = behaviorData.currentContacts || [];
```

Comment:
`behaviorData` is for properties.

Reply:
Yes no need to initialise from anything, just set it to `[]`.